app/src/main/java/com/github/houseorganizer/houseorganizer/LoginActivity.java

```diff
@@ -85,11 +85,31 @@ protected void onCreate(Bundle savedInstanceState) {
     );
 
     /**
-     *  Activate the Google Sign-In button for the user to authenticate
+     *  Activates the discover and Google Sign-In buttons for
+     *  the user to authenticate
      */
     private void activateSignInButton(){
```

Comment:
Maybe rename the function since its not actually activating anything

Reply:
Renamed it to `setUpSignInButtons`